UKHomeOffice/sgmr-service
# SPCR-466 Create PageContainer and move Voyage form pages to it

### AC / Description of changes

- create PageContainer to hold layout, ErrorSummary, LoadingSpinner (in preparation for handling page level errors)
- update pages to use new PageContainer
- - Voyage departure
- - Voyage arrival
- - Voyage pleasure craft selector (single)
- - Voyage pleasure craft selector (0 or multiple)
- - Voyage people selector
- - Voyage people manifest
- - Voyage employee check (is anyone employed)
- - Voyage employee select (select who is employed)
- - Voyage check your answers
- - Voyage submitted

### To Test (Developer)
- create a voyage
- click submit on each page without entering any data
- > page should error
- complete form
- submit
- > page should submit


### Notes

**Not done yet**

- Pleasure craft page
- Pleasure craft form 
- People page
- People form
- Account page
- Account form
- Dashboard
- Manage your voyage reports

---

**Remember to merge to feature branches not `master`.**
